doc/mgr/dashboard.rst

```diff
@@ -752,18 +753,18 @@ in order to manage silences.
 
     $ ceph dashboard set-prometheus-api-host 'http://localhost:9090'
 
-  After setting up the hosts, you have to refresh the dashboard in your browser window.
+  After setting up the hosts, refresh your browser's dashboard window/tab.
```

Comment:
```suggestion
  After setting up the hosts, refresh your browser's dashboard window or tab.

Reply:
On the one hand, this is a nitpick. On the other hand, I concur with it because I prefer saying "window or tab" to "window slash tab".